lib/arclight/traject/ead2_config.rb

```diff
@@ -249,9 +253,46 @@
     accumulator << context.output_hash['level_ssm']&.map(&:capitalize)
   end
 
+  to_field 'accessrestrict_ssm', extract_xpath('./xmlns:accessrestrict/xmlns:p')
+
+  to_field 'parent_access_restrict_ssm', extract_xpath('./xmlns:accessrestrict/xmlns:p')
+
+  to_field 'parent_access_restrict_ssm' do |_record, accumulator, context|
+    next unless context.output_hash['accessrestrict_ssm'].nil?
+
+    context.output_hash['parent_ssm']&.each do |id|
+      accumulator.concat Array
+        .wrap(context.clipboard[:parent]&.output_hash&.[]('components'))
```

Comment:
I'm pretty sure that `clipboard[:parent]` is just always the top-level document. Is that ever going to be a problem? Is there a different or better way to get the direct parent if that's what's needed?

Same question/answer will apply to lines 273, 285, and 293.